.github/scripts/get-release-timelines.sh

```diff
@@ -0,0 +1,88 @@
+
+release_branch="Version-v${VERSION}"
+release_pr_title="Version v${VERSION}"
+
+release_pr=$(gh pr list --repo "${OWNER}/${REPOSITORY}" --head "${release_branch}" --base master --state merged --json title,mergedAt | jq --arg title "${release_pr_title}" '.[] | select(.title == $title)')
+release_pr_merged_at=$(echo "${release_pr}" | jq -r '.mergedAt')
+
+if [[ -z "${release_pr_merged_at}" || "${release_pr_merged_at}" == "null" ]]; then
+  echo "::error::Release PR with title '${release_pr_title}' not found"
+  exit 1
+fi
+
+rollout_1_at="null"
+rollout_10_at="null"
+rollout_100_at="null"
+
+runway_release_id="${RUNWAY_APP_ID}:${VERSION}"
+release_submitted_at=$(curl --silent --header "X-API-Key: ${RUNWAY_API_KEY}" "https://api.runway.team/v1/app/${RUNWAY_APP_ID}/release/${runway_release_id}" | jq -r '.submittedAt')
+
+release_label="regression-RC-${VERSION}"
+release_blockers=$(gh issue list --repo "${OWNER}/${REPOSITORY}" --state all --label "release-blocker,${release_label}" --limit 100 --json number,createdAt)
+rollout_blockers=$(gh issue list --repo "${OWNER}/${REPOSITORY}" --state all --label "rollout-blocker,${release_label}" --limit 100 --json number,createdAt)
+all_blockers=$(jq -s 'add' <<< "${release_blockers} ${rollout_blockers}")
+
+echo "${all_blockers}" | jq -c '.[]' | while IFS= read -r issue; do
+  issue_created_at=$(echo "${issue}" | jq -r '.createdAt')
+  issue_number=$(echo "${issue}" | jq -r '.number')
+
+  timeline=$(gh api --header "Accept: application/vnd.github+json" --header "X-GitHub-Api-Version: 2022-11-28" "/repos/${OWNER}/${REPOSITORY}/issues/${issue_number}/timeline")
+  last_team_assigned_at=$(echo "${timeline}" | jq -r '[.[] | select(.event == "labeled" and (.label.name | startswith("team-")))] | last.created_at')
+  triage_completed_at=$(echo "${timeline}" | jq -r '[.[] | select(.event == "unlabeled" and (.label.name == "needs-triage"))] | last.created_at')
+
+  # shellcheck disable=SC2016
+  linked_prs=$(gh api graphql -F owner="${OWNER}" -F name="${REPOSITORY}" -F number="${issue_number}" -f query='
+    query($owner: String!, $name: String!, $number: Int!) {
+      repository(owner: $owner, name: $name) {
+        issue(number: $number) {
+          closedByPullRequestsReferences(first: 100, includeClosedPrs: true) {
+            nodes {
+              title
+              createdAt
+              mergedAt
+            }
+          }
+        }
+      }
+    }' | jq -r '.data.repository.issue.closedByPullRequestsReferences.nodes')
+
+  bugfix_pr=$(echo "${linked_prs}" | jq -r '[.[] | select(.title | startswith("chore(runway): cherry-pick") | not)] | last')
+  bugfix_pr_created_at=$(echo "${bugfix_pr}" | jq -r '.createdAt')
+  bugfix_pr_merged_at=$(echo "${bugfix_pr}" | jq -r '.mergedAt')
+
+  cherry_pick_pr=$(echo "${linked_prs}" | jq -r '[.[] | select(.title | startswith("chore(runway): cherry-pick"))] | last')
```

Comment:
Sometimes developers forget to do the cherry-pick with runway. Also, sometimes they have to do it manually from the command line because of merge conflicts. See the list of cherry-pick PRs: https://github.com/MetaMask/metamask-extension/pulls?q=is%3Apr+cherry+is%3Aclosed

Could we update this script so that it matches any PR with "cherry-pick" in the title, not just the "chore(runway) PRs"?